Cargo.toml

```diff
@@ -10,6 +10,7 @@ members = [
     "rusk-abi",
     "rusk-schema",
     "rusk",
+    "node-common",
```

Comment:
If that is the case, I'd suggest we should find a more appropriate name now and rename it within this PR/issue,  instead of reworking it after both node and consensus are already importing the crate.

Any suggestions @herr-seppia @ZER0

Reply:
What do you think about node-data or node-structure?

Reply:
I'll rename it to `node-data` after we merge https://github.com/dusk-network/rusk/pull/849

Reply:
Both Merging and renaming have been done